docs/addsegment-guide.md

```diff
@@ -37,12 +37,12 @@ In the above command user can add the existing data to the carbon table as a new
 During add segment, it will infer the schema from data and validates the schema against the carbon table. 
 If the schema doesn’t match it throws an exception.
 
-###Changes to tablestatus file
+### Changes to tablestatus file
```

Comment:
Please change line 441 ‘format’  to `format`

Reply:
done